apps/examples/board_specific_demo/Kconfig

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
Only some files have the under descriptions, some are not.
Could you add if needed?
```
# For a description of the syntax of this configuration file,
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
#
```